### PR TITLE
Improve the initial setup by adding the glibc compatability layer

### PR DESCRIPTION
**Pull request by ghost**
1. Remove webpack from the deps of app (docker-compose)
2. Add the glibc compatability layer (for m1)
3. Handle empty fetch of CAMP_START / CAMP_END

### CODE REVIEW

File: config/initializers/constants.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module Orca
-  CAMP_START = Date.parse(ENV.fetch('CAMP_START', '2022-07-23'))
-  CAMP_END = Date.parse(ENV.fetch('CAMP_END', '2022-08-06'))
+  CAMP_START = Date.parse(ENV.fetch('CAMP_START').presence || '2022-07-23')
```
Comment (ghost) [inline config/initializers/constants.rb]: Old version would fail if  CAMP_START eq ""

File: docker-compose.yml [hunk 2]
```diff
@@ -52,7 +52,6 @@ services:
       - db
       - mail
       - memcached
-      - webpack
```
Comment (ghost) [inline docker-compose.yml:57]: webpack service was removed during the upgrade to 3.1.1

File: config/initializers/constants.rb (hunk 1 above)
Comment (diegosteiner) [reply config/initializers/constants.rb]: ENV.fetch will fail with exception if key does not exists, so the default behavior is broken here

Comment (diegosteiner) [reply config/initializers/constants.rb]:
```
CAMP_START = Date.parse(ENV['CAMP_START'].presence || '2022-07-23')
```
would fix this

**Review by diegosteiner — CHANGES_REQUESTED** [review]

**Pull request merged**